### DISCUSSION TITLE
SpiralLinux 11.220606 release

### DISCUSSION THREAD

**Discussion by geckolinux** [SpiralLinux/SpiralLinux-project#6] [Announcements]
![SpiralLinux-transparent-small](https://user-images.githubusercontent.com/2692138/172185018-77411974-5e2b-43b6-aed8-42f492047cd6.png)

The SpiralLinux project is pleased to announce the initial 11.220606 release with 7 editions specifically tailored to each of the major Linux desktop environments available in the Debian repositories:

- Cinnamon [(screenshot)](https://raw.githubusercontent.com/SpiralLinux/spirallinux.github.io/main/images/screenshots/screenshot-Cinnamon.png)
- XFCE [(screenshot)](https://raw.githubusercontent.com/SpiralLinux/spirallinux.github.io/main/images/screenshots/screenshot-XFCE.png)
- Gnome [(screenshot)](https://raw.githubusercontent.com/SpiralLinux/spirallinux.github.io/main/images/screenshots/screenshot-Gnome.png)
- Plasma / KDE [(screenshot)](https://raw.githubusercontent.com/SpiralLinux/spirallinux.github.io/main/images/screenshots/screenshot-Plasma.png)
- Mate [(screenshot)](https://raw.githubusercontent.com/SpiralLinux/spirallinux.github.io/main/images/screenshots/screenshot-Mate.png)
- Budgie [(screenshot)](https://raw.githubusercontent.com/SpiralLinux/spirallinux.github.io/main/images/screenshots/screenshot-Budgie.png)
- LXQt [(screenshot)](https://raw.githubusercontent.com/SpiralLinux/spirallinux.github.io/main/images/screenshots/screenshot-LXQt.png)

SpiralLinux is a selection of Linux spins built from Debian GNU/Linux, with a focus on simplicity and out-of-the-box usability across all the major desktop environments. SpiralLinux serves as an alternative live installation method for a highly reliable customized Debian system using only official Debian package repositories.

## SpiralLinux features

- Installable live DVD / USB images around 2GB in size and carefully configured for a wide array of popular desktop environments
- Built from Debian Stable packages with newer hardware support preinstalled from Debian Backports
- Easily upgradable to Debian's Testing or Unstable branches with just a few clicks ([instructions](https://github.com/SpiralLinux/SpiralLinux-project/wiki#switching-from-debian-stable-to-the-testing-or-unstable-branch))
- Optimal Btrfs subvolume layout with Zstd transparent compression and automatic Snapper snapshots bootable via GRUB for easy rollbacks ([instructions](https://github.com/SpiralLinux/SpiralLinux-project/wiki#working-with-btrfs-snapshots-and-rollbacks))
- Graphical manager for Flatpak packages and preconfigured Flatpak theming
- Font rendering and color theming preconfigured for optimal legibility
- Preinstalled proprietary media codecs and non-free Debian package repositories ready to use
- Broad hardware support with a wide array of proprietary firmware preinstalled
- Extensive printer support with relaxed permissions for printer administration
- Optimal power management with TLP preinstalled
- VirtualBox support available out-of-the-box
- Enables zRAM swap by default for better performance on low-end hardware
- Normal users can operate and administer the system without recurring to the terminal
- Depends entirely on the Debian infrastructure, thus avoiding the "developer-hit-by-a-bus" concern
- Installed system can be smoothly upgraded to future Debian releases while retaining its unique SpiralLinux configuration

## Why another Debian-based distro?

Great question. Debian does an excellent job of providing the fundamentals needed for a highly reliable and flexible operating system. However, it is a sprawling and impersonal organization that does not focus on being friendly for the end-user. So a plethora of Debian-based distributions have arisen with the objective of adding a layer of polish, albeit with varying degrees of success. Most of these efforts depend on a single developer or a small group of developers with far fewer resources than the official Debian project. Many such projects create custom packages and supplemental package repositories that will cease to exist and leave their users stranded in the event that the project ever ceases to function. SpiralLinux, on the other hand, has been carefully designed to be _completely_ dependent on the extensive development resources and excellent package infrastructure of the upstream Debian project. Debian itself provides a base system that is capable of being very user-friendly when properly configured. This is where SpiralLinux comes into play. Great effort has been expended in polishing the SpiralLinux default configuration for all the major desktop environments using the packages and mechanisms that Debian itself provides. So in effect, a SpiralLinux installation is actually a legitimate Debian installation that can be infinitely upgraded from the official Debian repositories while retaining its unique SpiralLinux configuration.

## Who are you?

I've used Linux on the desktop since approximately 2001, with Debian-based distributions playing a key role in that experience. I have always appreciated Debian's extreme flexibility, reliability, speed, and excellent core infrastructure. But for desktop usage, Debian's lack of polish and geeky nature always led me to avoid it for myself and other users that I support. This projects sets out to change that. I am also the creator and maintainer of the [GeckoLinux](http://geckolinux.github.io/) set of spins built from openSUSE.

## Download SpiralLinux: https://spirallinux.github.io

<hr /><em>This discussion was created from the release <a href='https://github.com/SpiralLinux/SpiralLinux-project/releases/tag/11.220606'>SpiralLinux 11.220606 release</a>.</em>

**Comment by Xispra** [comment]
I saw the review of SpiralLinux on [9-5 Linux](https://9to5linux.com/meet-spirallinux-a-debian-based-distro-focused-on-simplicity-and-usability) yesterday and downloaded and installed the plasma version today. Bravo. I'm really floored. Very sensible defaults, pure Debian, but with a great, very fast Calamares installer. And the live USB ISO image booted on my new system in just seconds, I'm very impressed. 

I like to encrypt my systems with LUKS, but couldn't figure out how to do it with the netinst Debian installer without allowing the installer to automatically partition my drive. Unfortunately, the standard Debian installer insists on creating a 500 MB unencrypted ext2 /boot partition, and there doesn't seem to be any way to adjust that. The problem is not just the security implications of an unencrypted /boot partition, but 500 MB just isn't big enough. I installed my previous Debian systems about 7 or 8 years ago when the /boot partition was 240 MB, which was fine at the time for a few kernels, but as I upgraded from Jessie to Bullseye over the years it is not longer even large enough for two kernels, so now it is a real struggle just to keep the systems updated. Even with only a single kernel I run out of room in /boot, but can still somehow boot to the new kernel and purge my previous kernel and unwedge apt. In another few years the new 500 MB /boot partitions would give me the same headaches as the kernel keeps expanding like my waist. And bugreports on this and related issues have all be gathering dust for years. Even the Ubuntu automatic LUKS partitioning scheme creates about a 1 TB /boot partition!  
![partman_choose_partition_0](https://user-images.githubusercontent.com/5503745/173223377-b3d5ba62-6fb1-41dd-a3ee-c170198446fa.png)



With the Calamares installer it just created a single 300 MB vfat /boot/efi partition and all the kernels are encrypted in the main body of the drive so I can keep as many as I want, heck, maybe even install a few headers & memtest+. 

I was surprised that Spirallinux isn't even on distrowatch yet, but it was just what I was looking for. Thank you very many, I'm very grateful.

**Reply by geckolinux** [reply]
Hi there @Xispra , thanks so much for the positive feedback, really made my day! The Calamares project definitely deserves a lot of credit for everything they've done to create an easy and flexible installer with sensible defaults. And once installation is out of the way I'm sure that the Debian base will take good care of you for many years to come.

SpiralLinux was just released this week, so Distrowatch hasn't approved it yet for listing, although it is listed under New Submissions. It looks like it could take many months or even years to get listed, but it might help if SpiralLinux users vote for it to be listed by clicking on the "Recommend" button.
